src/templates/icon.variables.liquid

```diff
@@ -211,3 +248,10 @@ Icons are order A-Z in their group, Solid, Outline, Thin (Pro only) and Brand
 @big: 2em;
 @huge: 4em;
 @massive: 8em;
+
+/* Duotone specifics */
+@duotonePrimaryColor: var(--fa-primary-color, inherit);
```

Comment:
This wont work unless one declares --fa-primary-color somewhere in their code.
What i meant was to set a default color (hex value) as default instead of a css variable (and --fa-primary-color is from the original FA lib which isnt defined in FUI....and..(even if not really relevant anymore) the var() doesnt work in IE/legacy Edge

Reply:
Ok so what should that default color be to fit with FUI?

Reply:
> Ok so what should that default color be to fit with FUI?

inherit should be sufficient i believe

Reply:
Alright, I will make the changes over the week-end. Thanks for you help